pkg/volume/rbd/rbd_test.go

```diff
@@ -385,7 +385,7 @@ func TestPlugin(t *testing.T) {
 					RBDPool:      "pool1",
 					RBDImage:     "image1",
 					FSType:       "ext4",
-					ReadOnly:     true,
+					ReadOnly:     false,
```

Comment:
This test was faulty, it attempted to mount a unformatted volume in a readonly mode which obviously fails. The recent changes into format detection code in mount library broke this.